lib/jekyll/commands/new.rb

```diff
@@ -114,6 +116,30 @@ def site_template
         def scaffold_path
           "_posts/0000-00-00-welcome-to-jekyll.markdown.erb"
         end
+
+        def bundle_install(path)
+          Jekyll::External.require_with_graceful_fail "bundler"
+          $stdout.print "\nRun 'bundle install' with default theme? (Y/N)  "
+          @ans = ENV["CI"] ? "N" : $stdin.gets.chomp
+          until @ans == "Y" || @ans == "N"
+            $stdout.print "Invalid input. Input 'Y' or 'N'  "
+            @ans = $stdin.gets.chomp
+          end
+          stdinput(path)
+        end
+
+        def stdinput(path)
+          if @ans == "Y"
+            Jekyll.logger.info "Running bundle install in", path.to_s + "..."
+            Dir.chdir(path.to_s) do
+              system("bundle install")
+            end
+          elsif @ans == "N"
+            Jekyll.logger.warn "\nBundle install skipped.\n" \
+            "Edit your Gemfile and run 'bundle install' " \
```

Comment:
Technically they won't be able to do anything Jekyll until they bundle install.  This should be obvious though I will concede to new users it might not be obvious, so I don't disagree if you choose to leave it because I can see it's benefits.